server.js

```diff
@@ -1,9 +1,18 @@
 require ('dotenv').config()
 const express = require("express");
-const app = express();
+const {route} = require('./routes/admins');
 require('./database/database');
 
+const app = express();
+const {notFound} = require('./middleware/not_found');
+const errorHandlerMiddleware = require('./middleware/errorHandler');
+
 app.use(express());
+app.use('/', route);
+app.use(notFound);
+app.use(errorHandlerMiddleware);
+
+
 const PORT = 7000;
```

Comment:
We don't need two empty lines
```suggestion
app.use(errorHandlerMiddleware);

const PORT = 7000;
```